site/faq.md

```diff
@@ -227,3 +227,13 @@ A live example of this can be seen
 Please note that [flux#749](https://github.com/weaveworks/flux/issues/749)
 limits this currently to resources of the type `namespace`, `service`
 and `deployment`.
+
+To stop ignoring deployments, you currently need to make sure
+the annotations are removed on the cluster level as well. This is
+Because of a current [limitation in
+Flux](https://github.com/weaveworks/flux/issues/1211). The easiest
+way to do this is to run:
```

Comment:
This is a bit misleading. If you _only_ annotated the resource in git, then flux will ignore it (and hence also not add the annotation to the cluster), and stop ignoring it when the annotation is removed.